### PR TITLE
Added Plot Options to Bar and Pie Chart dialog

### PR DESCRIPTION
**Pull request by derekagorhom**
fixes partly #7556 
Replace #7810 

@N-thony , @rdstern this PR is ready for review

### CODE REVIEW

**Review by rdstern — DISMISSED** [review]
@derekagorhom this looks fine.  I assume the old one will also be closed?  Now over to @lloyddewit

File: instat/dlgBarAndPieChart.vb [hunk 1]
```diff
@@ -93,7 +102,7 @@ Public Class dlgBarAndPieChart
         bReset = False
         TestOkEnabled()
         autoTranslate(Me)
-        ChangeButtonsText()
+        'ChangeButtonsText()
```
Comment (lloyddewit) [inline instat/dlgBarAndPieChart.vb:105]:
```suggestion
```

File: instat/dlgBarAndPieChart.vb [hunk 2]
```diff
@@ -655,56 +671,75 @@ Public Class dlgBarAndPieChart
...
+                                clsNewThemeFunction:=clsThemeFuction, dctNewThemeFunctions:=dctThemeFunctions, clsNewGlobalAesFunction:=clsLabelAesFunction, ucrNewBaseSelector:=ucrBarChartSelector,
+                                clsNewCoordPolarFunction:=clsCoordPolarFunction, clsNewCoordPolarStartOperator:=clsCoordPolarStartOperator, clsNewXScaleDateFunction:=clsXScaleDateFunction, clsNewAnnotateFunction:=clsAnnotateFunction,
+                                clsNewScaleFillViridisFunction:=clsScaleFillViridisFunction, clsNewScaleColourViridisFunction:=clsScaleColourViridisFunction, clsNewYScaleDateFunction:=clsYScaleDateFunction,
+                                strMainDialogGeomParameterNames:=strGeomParameterNames, bReset:=bResetSubdialog)
         sdgPlots.ShowDialog()
         bResetSubdialog = False
-        'Warning, when coordinate flip is added to coordinates tab on sdgPLots, then link with ucrChkFlipCoordinates...
+    End Sub
+    Private Sub toolStripMenuItemBarchartOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemBarchartOptions.Click
```
Comment (lloyddewit) [inline instat/dlgBarAndPieChart.vb:724]:
```suggestion

    Private Sub toolStripMenuItemBarchartOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemBarchartOptions.Click
```

File: instat/dlgBarAndPieChart.vb [hunk 3]
```diff
@@ -655,56 +671,75 @@ Public Class dlgBarAndPieChart
...
-            sdgLayerOptions.ShowDialog()
-            bResetBarLayerSubdialog = False
-        End If
+    Private Sub toolStripMenuItemWordcloudOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemWordcloudOptions.Click
+        openSdgLayerOptions(clsGeomTextWordcloudFunction)
     End Sub
 
+    Private Sub toolStripMenuItemPointOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemPointOptions.Click
+        openSdgLayerOptions(clsPointsFunction)
+    End Sub
```
Comment (lloyddewit) [inline instat/dlgBarAndPieChart.vb:742]:
```suggestion
    End Sub
    
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Pull request merged**